### PR TITLE
OCTOPUS-602: Changes added for Customization script

### PR DESCRIPTION
**Pull request by Chandan-Abhyankar**
OCTOPUS-602: Changes added for Customization script

### CODE REVIEW

**Comment by Chandan-Abhyankar** [comment]
Raising a PR for Customization Script changes

File: openstack/intel-worker/template/worker-amd64.ign.j2
Comment (prb112) [inline openstack/intel-worker/template/worker-amd64.ign.j2:1]: let's delete this one file

Comment (Chandan-Abhyankar) [reply openstack/intel-worker/template/worker-amd64.ign.j2:1]: Yes, in my PR code , I have deleted 'template' folder and created 'templates' folder with my file inside it.

**Review by prb112 — APPROVED** [review]

**Pull request merged**